src/Microsoft.VisualStudio.ProjectSystem.Managed.VS.UnitTests/ProjectSystem/VS/RenamerTests.cs

```diff
@@ -20,10 +22,38 @@ public class RenamerTests
         [InlineData("partial class Foo {} partial class Foo {}", "Foo.cs", "Bar.cs")]
         [InlineData("struct Foo { decimal price; string title; string author;}", "Foo.cs", "Bar.cs" )]
         [InlineData("enum Foo { None, enum1, enum2, enum3, enum4 };", "Foo.cs", "Bar.cs")]
+        [InlineData("namespace n1 {class Foo{}} namespace n2 {class Foo{}}", "Foo.cs", "Bar.cs")]
+        public async Task Rename_Symbol_Should_TriggerUserConfirmationAsync(string soureCode, string oldFilePath, string newFilePath)
+        {
+            var userNotificationServices = IUserNotificationServicesFactory.Implement(f =>
```

Comment:
The Moq framework records all the calls that are made on the mocked object. So you can simply do Mock.Get(userNotificationServices).Verify(f => f.Rename()) to get the Mock object from the instance and verify that the Rename method was called.